frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Firehose } from '@console/internal/components/utils';
+import { StorageClassDropdownInner } from '@console/internal/components/utils/storage-class-dropdown';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+const cephStorageProvisioners = ['ceph.rook.io/block', 'cephfs.csi.ceph.com', 'rbd.csi.ceph.com'];
+
+export const OCSStorageClassDropdown: React.FC<OCSStorageClassDropdownProps> = (props) => (
+  <Firehose resources={[{ kind: 'StorageClass', prop: 'StorageClass', isList: true }]}>
+    <StorageClassDropdown {...props} />
+  </Firehose>
+);
+
+const StorageClassDropdown = (props) => {
+  const getFilteredStorageClassData = (storageClassData: K8sResourceKind[]): K8sResourceKind[] =>
+    storageClassData.filter(
+      (sc: K8sResourceKind) =>
+        !(
```

Comment:
I'm confused: are we taking the ceph provider *out* of the array? It looks like you're remove them.

Reply:
@spadgett, we have to show the provides which are not ceph based, so we are creating another array `getFilteredStorageClassData` with storageClasses other than ceph based.